roles/wildfly_systemd/templates/wfly.conf.j2

```diff
@@ -1,7 +1,9 @@
-WILDFLY_SERVER_CONFIG={{ instance_name }}.xml
-
-WILDFLY_OPTS=-Djboss.server.config.dir={{ wildfly_systemd.home }}/standalone/configuration/ -Djboss.server.base.dir={{ basedir }} -Djboss.tx.node.id={{ instance_name }} {% if instance_id is defined %}-Djboss.socket.binding.port-offset={{ 100 * (instance_id - wildfly_port_range_offset) }}{% endif %} -Djboss.node.name={{ instance_name }}
+# {{ ansible_managed }}
```

Comment:
Should not this commented line be removed completely?

Reply:
that is actually a (good practice) ansible feature, we should check it present on any template we deploy
https://docs.ansible.com/ansible/latest/reference_appendices/config.html#default-managed-str